Source/Extensions/Blazorise.Icons.Bootstrap/BootstrapIcons.cs

```diff
@@ -426,7 +426,6 @@ public static class BootstrapIcons
     public static readonly string CloudFog2 = "bi-cloud-fog2";
     public static readonly string CloudHailFill = "bi-cloud-hail-fill";
     public static readonly string CloudHail = "bi-cloud-hail";
-    public static readonly string CloudHaze1 = "bi-cloud-haze-1";
```

Comment:
@stsrki On purpose, apparently removal of nonexisting icons. https://github.com/twbs/icons/commit/a946ccef98c67a5c740833235a76e9bb64c09d54

Reply:
In that case, it might be better to target `master` branch to be released with the coming 1.4. That way, we would not break the stable 1.3.x, and we can note in the release notes why the change is done.